### ISSUE TITLE
ObserveInBackground is not defined

### ISSUE THREAD

**Issue by hugoj-goncalves** [RaiMan/SikuliX1#171]
using version 1.1.4 build 325

`[error] NameError ( name 'observeInBackground' is not defined )`

did it get renamed or something?

**Comment by balmma** [comment]
Can you please provide a not working example script?

**Comment by balmma** [comment]
Was it working before the 325 Build?

**Comment by balmma** [comment]
Do you call it like SCREEN.observeInBackground()?

**Comment by balmma** [comment]
For your reference a working example of observeInBackground on the primary screen:
```
def errorHandler(event):
    print(event)
    event.getMatch().click()
    type("hello world"+Key.ENTER)
    event.repeat(2)

Settings.ObserveScanRate = 1

SCREEN.onAppear("1563559902737.png", errorHandler)

SCREEN.observeInBackground()

try:
    sleep(20)

finally:
    SCREEN.stopObserver()

```

**Comment by balmma** [comment]
@RaiMan 
Should this also work as undotted function in Jython and use SCREEN by default?

**Comment by hugoj-goncalves** [comment]
Oh that's it.
I was trying without the SCREEN region thing.
Reading the docs is a little confusing that _observe()_ works without a region and _observeInBackground()_ doens't.
And I had set a mental note to test with a region and completly forgot also.
Thanks and sorry for bothering.

**Comment by RaiMan** [comment]
@balmma 
@hugoj-goncalves 
 - `observeInBackground()` undotted use in Python scripts
 ... should be bound to the `SCREEN` object as it is with `observe()`(fixed in build #329)
 - the default is `FOREVER` when using `observe()` and `observeInBackground()` without parameter (constant `FOREVER` simply is the largest float/double number available in that environment)
 - though annotated as deprecated `observe(someTime, true)` is still available being the same as the now preferred `observeInBackground(someTime)`
 - I will revise the docs, to be more precise here
